ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bfd.schema.yml

```diff
@@ -32,6 +32,11 @@ keys:
             type: int
             min: 3
             max: 50
+      session_snapshot_interval:
```

Comment:
Would it be a good idea to instead have 2 separate <int> type variables with the schema enforcing min/max values?

session_snapshot_interval:
  type: int
  min: 10
  max: 3600
session_snapshot_interval_dangerous:
  type: int
  min: 1
  max: 9

Reply:
We have evaluated that as well, and we will stick with the implementation already done in this PR.